### PR TITLE
WEBAPP-1: Redesign Header

### PR DESCRIPTION
**Pull request by Schedulaar**
This is a preview. Please check if you're content with the header. I think I'll extract the swiping part (between extras, categories and events) of the header to another task, since it'll involve some more complexity, Because we need to have multiple 'views' loaded at the same time (categories page should stay at the exact same page when swiping to the events view and therefore changing the url to the events url, then swiping back to the categories view)

### CODE REVIEW

File: src/components/RichLayout/Header.js [hunk 1]
```diff
@@ -1,87 +1,94 @@
...
           </div>
+          {
+            this.isMenuEnabled() &&
+            <div className={style.menuItems}>
+              { this.isExtrasEnabled() && <Link href={this.props.navigation.home}>{t('common:extras')}</Link> }
+              <Link href={this.props.navigation.home}>{t('common:categories')}</Link>
+              { this.isEventsEnabled() && <Link href={this.props.navigation.events}>{t('common:news')}</Link> }
+            </div>
+          }
         </div>
```
Comment (Schedulaar) [inline src/components/RichLayout/Header.js]: I need to have some logic, to detect wether Categories, Extras or Events is currently shown, but I am unsure how to do it. Do I really need to get router.path and check the url? 🤔

Comment (maxammann) [reply src/components/RichLayout/Header.js]: https://github.com/FormidableLabs/redux-little-router#link
You can do:
<Link
  href='/wat'
  className='normal-link' 
  activeProps={{ className: 'active-link' }}
>
  Wat
</Link>`

:)
`

File: locales/ar/common.json [hunk 2]
```diff
@@ -1,6 +1,9 @@
 {
   "location": "الموقع",
   "date": "تاريخ",
+  "extras": "إضافات",
+  "categories": "الفئات",
+  "news": "الأخبار",
```
Comment (maxammann) [inline locales/ar/common.json]: Are those professional translations?

Comment (maxammann) [reply locales/ar/common.json]: Farsi translations are missing :)

Comment (Schedulaar) [reply locales/ar/common.json]: Yep, These are the constants sheet, Farsi is missing there as well

File: package.json [hunk 3]
```diff
@@ -91,6 +91,7 @@
     "postcss-selector-matches": "^3.0.1",
     "postcss-selector-not": "^3.0.1",
     "prop-types": "^15.6.0",
+    "react-headroom": "^2.2.2",
```
Comment (maxammann) [inline package.json]: +1 for this :D let's see how this behaves :D

File: locales/ar/common.json (hunk 2 above)
Comment (maxammann) [reply locales/ar/common.json]: Farsi has been added recently, french aswell :)

File: src/components/RichLayout/Header.js (hunk 1 above)
Comment (Schedulaar) [reply src/components/RichLayout/Header.js]: I think the active class will only take effect, if the href *exactly* matches the current url (so if the url is `/augsburg/de/welcome`, then the active-class will be removed, but I still want the `Categories` to be printed bold.) But I'll give it a try...

File: locales/ar/common.json (hunk 2 above)
Comment (Schedulaar) [reply locales/ar/common.json]: Farsi  is done, french will be added in WEBAPP-133

**Pull request merged**